internal/pkg/log/zap/zap.go

```diff
@@ -0,0 +1,126 @@
+// Package zap sets up a logger using the Zap logging framework.
```

Comment:
Can we make this comment more explicit? Stating that this package implements the logging interface with the Zap Framework as its implementation?

Reply:
Discussed more about this here https://github.com/envoyproxy/xds-relay/pull/35#discussion_r398789804, I'll update the comment.